### ISSUE TITLE
Will it support netstandard2.0 in the future?

### ISSUE THREAD

**Issue by lintao185** [Sergio0694/ComputeSharp#839]

**Comment by Sergio0694** [comment]
No, there are no plans to add .NET Standard 2.0 support back. If you need it, you can use version 2.1.0 (or 2.2.0-preview1, if you want the new source generator attributes as well), which is still available on NuGet. That has support for .NET Standard 2.0 as well, and you can see there's unit tests on .NET Framework and UWP as well on that branch.
